### DISCUSSION TITLE
Signac pmbc<-TSSEnrichment(object = pbmc, fast=FALSE) met a problem

### DISCUSSION THREAD

**Discussion by wenzhong4** [stuart-lab/signac#1438] [General]
Signac is not working in R 4.3.1 ???

I run PBMC scATAC-seq in my local pc following the Vignettes Human PBMCs
 Seurat, Signac are currently versions. TSSEnrichment function met a problem as below:

> pbmc<-NucleosomeSignal(object = pbmc)
Found 8728 cell barcodes
Done Processing 43 million lines    
              
> pmbc<-TSSEnrichment(object = pbmc, fast=FALSE)
Extracting TSS positions
Finding + strand cut sites
[W::hts_idx_load3] The index file is older than the data file: C:\Users\wenzhong\Documents\722019_doc\New_folder\wenzhong\R_function\scATAC-seq-signac\atac_v1_pbmc_10k_fragments.tsv.gz.tbi

**Comment by realmungmung** [comment]
Did you figure it out? I also met error when running this code:
Error in slot(object = object, name = layer) : 
  argument "slot" is missing, with no default

**Reply by jasonyklam** [reply]
I'm getting the same error message too!

**Reply by realmungmung** [reply]
I figured it out you can try:
remotes::install_github("stuart-lab/signac", ref="develop")

**Reply by jasonyklam** [reply]
It seems to have worked! Thanks!
